volatility3/framework/plugins/linux/ip.py

```diff
@@ -127,7 +150,7 @@ def _gather_net_dev_link_info(self, net_device):
         ]
         flags_str = ",".join(flags_list)
 
-        yield net_ns_id, iface_name, mac_addr, operational_state, mtu, qdisc_name, qlen, flags_str
+        yield net_ns_id or renderers.NotAvailableValue(), iface_name, mac_addr, operational_state, mtu, qdisc_name or renderers.NotAvailableValue(), qlen, flags_str
```

Comment:
I mean, this is technically an internal function (because it's not a `classmethod`) so I'll let you off, but if people try and copy this for classmethods, I will point them at this comment and probably shake my finger and tut...  5:P